netmanager-app/core/apis/analytics.ts

```
@@ -0,0 +1,16 @@
+
+const axiosInstance = createAxiosInstance();
+
+export const dataExport = async (data: any) => {
+    try {
+        const headers = {
+            service: 'data-export'
+        }
+        const exportData =  axiosInstance.post(`${ANALYTICS_MGT_URL}/data-download`, data, { headers });
+        return exportData;
+    } catch (error) {
+        throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)